Communication/IO.cs

```diff
@@ -23,18 +23,34 @@ public static bool AnotherContest()
             return (ans == 'y');
         }
 
-        public static void DisplayContest(ContestDefinition contest)
+        public static void DisplayContest(ContestDefinition contest, bool gym)
         {
-            Console.WriteLine($"Id = {contest.Id} || Name = {contest.Name}");
+            Console.WriteLine($"\nId   = {contest.Id}");
+            Console.WriteLine($"Name = {contest.Name}");
```

Comment:
```suggestion
           Console.WriteLine($"Id = {contest.Id} || Name = {contest.Name}");
```

I think that it would be better this way